pandas/core/arrays/datetimelike.py

```diff
@@ -468,6 +543,67 @@ def _values_for_factorize(self):
     def _from_factorized(cls, values, original):
         return cls(values, dtype=original.dtype)
 
+    def _values_for_argsort(self):
+        return self._data
+
+    # ------------------------------------------------------------------
+    # Additional array methods
+    #  These are not part of the EA API, but we implement them because
+    #  pandas assumes they're there.
+
+    def searchsorted(self, v, side='left', sorter=None):
```

Comment:
Just to be clear, the name for the positional argument hasn't been finalized yet. We figuring out the relative value of matching NumPy vs. matching the rest of pandas here.

Reply:
well, as I have said multiple times, we are already using ``value`` in pandas, so -1 unless its that.

Reply:
Apparently Jeff has decided :)

Reply:
Will revert